TFlippy_TerritoryControl_Structures_Dev/Entities/Structures/Tavern/Tavern.as

```diff
@@ -102,6 +104,12 @@ void GetButtonsFor(CBlob@ this, CBlob@ caller)
 
 		CButton@ buttonWrite = caller.CreateGenericButton("$icon_paper$", Vec2f(0, -8), this, this.getCommandID("write"), "Rename", params);
 	}
+
+	//toggle tavern light
+	if(player !is null && player.getUsername() == this.get_string("Owner"))
```

Comment:
space between if and (

Reply:
wait a second the above if not written by me also has no space, ill add one there too then